src/script/components/report-card.ts

```diff
@@ -6,15 +6,19 @@ import {
   property,
   internalProperty,
 } from 'lit-element';
-import { RawTestResult } from '../utils/interfaces';
+import { RawTestResult, ScoreEvent } from '../utils/interfaces';
 
-import { largeBreakPoint, xLargeBreakPoint, smallBreakPoint } from '../utils/css/breakpoints';
+import {
+  largeBreakPoint,
+  xLargeBreakPoint,
+  smallBreakPoint,
+} from '../utils/css/breakpoints';
 
 import './score-results';
 
 @customElement('report-card')
 export class ReportCard extends LitElement {
-  @property() results: RawTestResult;
+  @property() results: RawTestResult | undefined;
```

Comment:
thinking of create a helper interface that is more semantic and keeps our data using only undefined, a lot of the issues I saw were because of explicit use of null, because null doesn't allow for optional chaining.

So proposal: 
1. use either values or undefined (x.x from a semantic standpoint ugh).
2. create a small type interface that abstracts this chaining e.g: `type Lazy<T> = T | undefined`
- usage: `Lazy<RawTestResult>`

Reply:
Im a fan of #2 👍🏻